eng/Dependencies.props

```diff
@@ -108,6 +108,7 @@ and are generated based on the last package release.
     <LatestPackageReference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.2.2" Version="$(MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion)" />
     <LatestPackageReference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x64" Version="$(MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion)" />
     <LatestPackageReference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x86" Version="$(MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion)" />
+    <LatestPackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DebugProxy" Version="$(MicrosoftAspNetCoreComponentsWebAssemblyRuntimePackageVersion)" />
```

Comment:
Have you thought about how the automation for this would work? I don't think there's a darc subscription to update dotnet/blazor@master -> dotnet/aspnetcore@master as yet and the runtime package is going away since wasm since ships as a .NET runtime pack (it's a SDK thing).